official/resnet/ctl/ctl_imagenet_benchmark.py

```diff
@@ -139,6 +139,24 @@ def benchmark_8_gpu(self):
     FLAGS.datasets_num_private_threads = 14
     self._run_and_report_benchmark()
 
+  def benchmark_8_gpu_amp(self):
+    """Test Keras model with eager, dist_strat and 8 GPUs with automatic mixed
+    precision.
+    """
+    self._setup()
+    FLAGS.num_gpus = 8
+    FLAGS.data_dir = self.data_dir
+    FLAGS.batch_size = 128 * 8
+    FLAGS.train_epochs = 90
+    FLAGS.epochs_between_evals = 10
+    FLAGS.model_dir = self._get_model_dir('benchmark_8_gpu_amp')
+    FLAGS.dtype = 'fp16'
+    FLAGS.fp16_implementation = 'graph_rewrite'
+    FLAGS.use_tf_function = True 
```

Comment:
This defaults to True, so no need to specify it. I don't like specifying it because then it's not clear this is exactly the same as `benchmark_8_gpu` except with AMP.

And same for the other benchmarks added.